webtransport/quic/handlers/client-indication.py

```diff
@@ -0,0 +1,26 @@
+import asyncio
```

Comment:
Let's call this file `client-indication.quic.py` in accordance with https://github.com/web-platform-tests/rfcs/blob/master/rfcs/quic.md

Reply:
Done.